app/src/main/java/org/simple/clinic/summary/linkId/LinkIdWithPatientSheetKey.kt

```diff
@@ -0,0 +1,22 @@
+package org.simple.clinic.summary.linkId
+
+import androidx.fragment.app.Fragment
+import kotlinx.android.parcel.Parcelize
+import org.simple.clinic.navigation.v2.ScreenKey
+import org.simple.clinic.patient.PatientUuid
+import org.simple.clinic.patient.businessid.Identifier
+
+@Parcelize
+data class LinkIdWithPatientSheetKey(
```

Comment:
I generally prefer to make the keys for the new screens nested classes in the same screen. Can we follow the same pattern here, please?